### PR TITLE
added /api/articles/article_id/comments endpont and associated testing

### PR DESCRIPTION
**Pull request by Cralus**
added the /api/articles/articles_id/comments endpoint for returning all comments associated with a certain article specified by id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -215,6 +215,41 @@ describe('GET /api/articles', () => {
...
+    }); 
+   test('should respond with a status of 200 if an article exists but has no comments', () => {
+        return request(app)
+        .get("/api/articles/10/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+        expect(comments).toEqual([]);
+        });
+    });
+})
```
Comment (elijaharley) [inline __tests__/app.test.js:260]: Nice work here Nathan, don't forget to test for 404 errors on this endpoint

**Review by elijaharley — CHANGES_REQUESTED** [review]
Great stuff. You just need to add a 404 test here Nathan and then you can merge

**Comment by Cralus** [comment]
Added handling for 404 errors if article id is a number but doesnt exist

**Review by k-huggs — COMMENTED** [review]
Nice work! Go ahead and merge

**Pull request merged**